src/Utils/resizer.ts

```diff
@@ -19,31 +19,11 @@ export const crop = (
     convert_to?: string
   }
 ) => {
-  const { width, height, quality, convert_to } = options
-
-  if (!src) return null
-
-  if (!width && !height) {
-    warn("requires width and height")
-    return src
-  } else if (width === 0) {
-    warn("width must be non-zero")
-    return src
-  } else if (height === 0) {
-    warn("height must be non-zero")
-    return src
-  }
-
-  const config = {
-    resize_to: "fill",
-    src,
-    width,
-    height,
-    quality: quality || 80,
-    convert_to,
```

Comment:
Best I can tell, there is no real use for this param in force. It's used in two places in the Collection to force a jpg. I have no idea why. Removing.